service/service.py

```diff
@@ -89,6 +89,15 @@ def internal_server_error(error):
 def get_suppliers(supplierID):
 	return supplier.find(supplierID)
 
+######################################################################
+# DELETE A SUPPLIER
+######################################################################
+
+@app.route('/suppliers/<string:supplierID>', methods = ['DELETE'])
+def delete_supplier(supplierID):
+    """ Route to delete a supplier """
+    return supplier.delete(supplierID)
```

Comment:
**supplier** is not defined.

Possible fix:
```
supplier= Supplier.find(supplierID)
if supplier:
    supplier.delete()
return make_response('', status.HTTP_204_NO_CONTENT)
```

Reply:
Fix added, thank you :)